src/styles/main.scss

```diff
@@ -0,0 +1,68 @@
+$stop-watch-size: 600px;
+$rod-size: 20px;
+$watch-rod-circle: 50%;
+$hands-indent: 50%;
+
+@mixin position-center {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+}
+
+@mixin position-transform-hands {
+  position: absolute;
+  left: $hands-indent;
+  transform-origin: bottom;
+}
+
+body {
+  margin: 0;
+}
+
+.stop-watch {
```

Comment:
According to BEM it's better to move this block to its own file
And the same for mixins and variables